sources/DBAL/SchemaMigrator.php

```diff
@@ -17,7 +17,10 @@ public function __construct(Connection $connection)
     public function migrateSchema(): void
     {
         $platform = $this->connection->getDatabasePlatform();
-        $fromSchema = $this->connection->getSchemaManager()->createSchema();
+        $fromSchema = method_exists(Connection::class, 'createSchemaManager')
```

Comment:
I think it is `doctrine/dbal`'s job to provide an upgrade path for this change.

For example, when `doctrine/dbal` introduced `fetchFirstColumn()`, several versions supported the new method and that is why I could create a simple PR like https://github.com/trompette/php-feature-toggles/pull/22 to remove the deprecation.

Reply:
Well, I agree that you are right as it is also the case for other functions that are depreciated and have no equivalent solution today... Let's close this PR!